### DISCUSSION TITLE
Decentralization and Token Distribution

### DISCUSSION THREAD

**Discussion by TheCryptoLawyers** [lex-node/SafeHarbor-X#6] [Ideas]
Governance tokens are designed to facilitate community governance and replace strictly hierarchical systems where there is designated, centralized management team, that is uniquely in control of and responsible for a venture's success. 

Decentralized governance is not an end in itself. The purpose of developing a decentralized system is to achieve some level of censorship resistance. 

If governance is not properly distributed among the community, the governance is not decentralized, and therefore unlikely to result in censorship resistance. 

Good policy should reward reward activity that is conducive to our industry's purpose, and punish activity that, in effect, hijacks the appearance of decentralization in order to circumvent the securities laws where they should apply. 

The above being said, I propose that we include an additional criteria (perhaps (a)(5), in order to qualify for this exemption. 

Specifically, to the extent that there is a centralized initial development team (which there almost always is), they should be unable to claim this exemption if they award themselves more than 10% of the total token supply. 

I realize that we should not be telling the industry how to innovate, and so this criterion is intended to be a reasonable ceiling for initial token allocations. If the team wants more tokens, they should lobby the community, and abstain from the vote, to earn them fairly. 10% is more than enough to have substantial influence, without creating a monopolistic level of control that can never be reasonably relinquished. 

I look forward to your thoughts, and I thank LexPunk for their continued contributions to the space.

**Comment by lex-node** [comment]
something like a cap on team ownership could make sense, but I am a bit reluctant to be so prescriptive as it could constrain creativity quite a bit...it's also a bit arbitrary...what if the team includes 100 people vs 5 people? 

there's also the issue that if further allocations are to be decided by the community and the project has an aggressive initial release schedule, then purely economic participants could have an outsized say in builder allocation, and they might be much more short-term oriented

we've seen this type of issue crop up recently with SushiSwap dev team suffering a lot of setbacks due to outsized user and token whale ownership and influence over the protocol equity, as compared to builders 

beyond ensuring lack of majority voting or economic control by the initial development team, I suspect it is best to let market forces be the disciplinarian here and give more room for the space to evolve--for example, if distribution mechanisms or governance mechanisms become more subtle and discriminating so that the governance quality of early recipients can be assured, then very low initial builder allocation with the rest in the hands of users might make sense, but at the moment what I see is that builders are one of the key types of stakeholders and that if they do not get a large enough allocation early enough then governance becomes *too* tilted in favor of users who, especially in the early days when there is a lack of "stickiness" to the network, might have a more short-term orientation

**Reply by lex-node** [reply]
an alternative idea would be to cap % ownership of any given member of the initial development team rather than capping the entire category....again, I hesitate to be prescriptive but directionally this sounds a bit better to me
